File: backend/src/main/java/com/tmk/vtcmanager/application/domain/recette/LigneRecetteFiltres.java
package com.tmk.vtcmanager.application.domain.recette;

import lombok.Builder;
import lombok.Data;

import java.time.LocalDate;

@Data
@Builder
public class LigneRecetteFiltres {
    private Long vehiculeId;
    private Long chauffeurId;
    private StatutLigneRecette statut;
    private LocalDate dateDebut;
    private LocalDate dateFin;
}

File: mobile/lib/features/recette/domain/entities/ligne_recette_filtres.dart
import 'ligne_recette.dart';

class LigneRecetteFiltres {
  final int? vehiculeId;
  final int? chauffeurId;
  final StatutLigneRecette? statut;
  final DateTime? dateDebut;
  final DateTime? dateFin;

  const LigneRecetteFiltres({
    this.vehiculeId,
    this.chauffeurId,
    this.statut,
    this.dateDebut,
    this.dateFin,
  });
}
